docs/documentation/release_notes/topics/26_4_0.adoc

```diff
@@ -50,3 +50,7 @@ It is now possible to change which identity providers are shown in the account c
 the `Show in Account console` setting. You can choose to show only those linked with a user or not show them at all.
 
 For more information, please see link:{adminguide_link}#_general-idp-config[General configuration] section in the {adminguide_name}.
+
+= Welcome Page changes
```

Comment:
I just realized, this should probably be a migration note, not a release note. It’s a functional behavior change, not a new feature.